plugins/woocommerce/bin/build-zip.sh

```diff
@@ -19,6 +19,7 @@ find "$PROJECT_PATH/assets/css/." ! -name '.gitkeep' -type f -exec rm -f {} + &&
 echo "Installing PHP and JS dependencies..."
 pnpm install
 echo "Running JS Build..."
+export NODE_ENV=production
```

Comment:
Note: I tried setting `NODE_ENV=production` higher up, but doing it before `pnpm` install caused the build to fail completely for me.

Reply:
@jonathansadowski This looks like an outdated version of the script, we also [set WOOCOMMERCE_BLOCKS_PHASE if its not set](https://github.com/woocommerce/woocommerce/blob/935364a6666ccf5d39627f7707578c6ab85eeb8d/plugins/woocommerce/bin/build-zip.sh#L8-L10) 

Can you please take the same approach because we use build-zip.sh to build dev versions too, ie when we build for live branches.

Reply:
Good call, just updated